c10/core/TensorImpl.cpp

```diff
@@ -188,16 +188,18 @@ void TensorImpl::HandleResize() {
   // If needed, we will free the data. the next mutable_data() call
```

Comment:
This one is legacy Caffe2, so better to just have a test at the top of the function to bail out if you're symbolic at all, in symmetry with the other legacy Caffe2 methods.

Reply:
did that but we still need to convert a SymInt to an int to simply keep a compiler happy about the types i.e. `numel_` is a SymInt now

Reply:
right, so test at the top, then do an assign to a local  int at the top and then use that in the body